akka-actor/src/main/scala/akka/dispatch/Dispatcher.scala

```diff
@@ -39,6 +39,8 @@ class Dispatcher(
 
   import configurator.prerequisites._
 
+  private val batchingEnabled = executorServiceFactoryProvider.isInstanceOf[NoBatchingExecutorFactoryProvider]
```

Comment:
shouldn't it be `!executorServiceFactoryProvider.isInstanceOf[NoBatchingExecutorFactoryProvider]` ?